### PR TITLE
Adding repo and area owners to the codeowners file

### PR DESCRIPTION
**Pull request by iamgusain**
Adding repo code and area owners as per the list from [here](https://microsoft.sharepoint.com/teams/ADAL/_layouts/15/Doc.aspx?sourcedoc={0339bdd6-b5e5-4d94-b4c9-ea47127f5023}&action=edit&wd=target%28Platforms%2FAndroid.one%7C28086b2a-f539-bc47-b229-1610fc61c903%2FCode%20Owners%7C83d87699-1bfc-46f7-b164-0df4c71a1d16%2F%29)

Once this change is merged, I will enable the branch protection setting to require codeowners approval

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,22 @@
...
+*       @AdamBJohnsonx @iambmelt @rpdome @shahzaibj @shoatman
+
+
+# Area Owners
+#---------------------
+
+# Automation
+/msalautomationapp/                                                                 @shahzaibj @rpdome
+/msal/src/test/                                                                     @shahzaibj @rpdome
+/msal/src/androidTest/                                                              @shahzaibj @rpdome
```
Comment (AdamBJohnsonx) [inline CODEOWNERS:16]: .github, we could talk about at some point.

**Review by AdamBJohnsonx — APPROVED** [review]

File: CODEOWNERS [hunk 2]
```diff
@@ -1,4 +1,22 @@
...
+
```
Comment (iambmelt) [inline CODEOWNERS:17]:
```suggestion
# Docs
/docs    @iambmelt @AdamBJohnsonx 
```

Comment (iambmelt) [inline CODEOWNERS:17]: # PackageInspector
/package-inspector    @iambmelt @shahzaibj

Comment (iambmelt) [inline CODEOWNERS:17]:
```suggestion
# PackageInspector
/package-inspector    @iambmelt @shahzaibj 
```

Comment (iambmelt) [inline CODEOWNERS:17]:
```suggestion
# PoP Benchmarker
/pop-benchmarker @iambmelt @AdamBJohnsonx 
```

**Review by iambmelt — APPROVED** [review]
A few suggestions, overall LGTM

**Pull request merged**